internal/pkg/places/interfaces.go

```diff
@@ -0,0 +1,13 @@
+package places
+
+import (
+	"TripAdvisor/internal/models"
+)
+
+type PlaceRepo interface {
+	GetPlaces() ([]models.Place, error)
+}
+
+type PlaceUsecase interface {
+	GetPlaces() ([]models.Place, error)
+}
```

Comment:
Надо добавить контексты во все методы usecase и repo
```suggestion
type PlaceRepo interface {
	GetPlaces(ctx context.Context) ([]models.Place, error)
}

type PlaceUsecase interface {
	GetPlaces(ctx context.Context) ([]models.Place, error)
}
```